samtranslator/model/eventsources/pull.py

```diff
@@ -8,16 +8,17 @@
 class PullEventSource(ResourceMacro):
     """Base class for pull event sources for SAM Functions.
 
-    The pull events are the streams--Kinesis and DynamoDB Streams. Both of these correspond to an EventSourceMapping in
-    Lambda, and require that the execution role be given to Kinesis or DynamoDB Streams, respectively.
+    The pull events are Kinesis Streams, DynamoDB Streams, and SQS Queues. All of these correspond to an EventSourceMapping in
+    Lambda, and require that the execution role be given to Kinesis Streams, DynamoDB Streams, or SQS Streams, respectively.
```

Comment:
I suppose we shouldn't be calling SQS as streams. SQS in itself should do. Thoughts ?

Reply:
Good catch; that was a miss. I refer to it as SQS Queues in the first sentence.